docs/nidcpower/class.rst

```diff
@@ -3430,7 +3470,7 @@ pulse_current_level_range
             +----------------+------------+
             | Permissions    | read-write |
             +----------------+------------+
-            | Channel Based  | True       |
+            | Channel Based  | Yes        |
```

Comment:
There are too many changes here that are independent from adding NISE. Can those occur in their own PR?

Reply:
See above about why.